### ISSUE TITLE
Inline script not allowed after uBO-refresh, but works with ctrl+F5

### ISSUE THREAD

**Issue by stonecrusher** [gorhill/uBlock#3525]
### Describe the issue
When trying to make a site work again (due to my standardly overblocking), I allow more and more scripts gradually and hit the big refresh button in between.
Now I came across an issue where an inline script is not executed when hitting the uBO refresh button, but is executed when refreshing with ctrl + F5.
This is confusing because I thought my rule was wrong / wouldn't work.

### Steps for anyone to reproduce the issue
1. Go to `https://www.mp3tag.de/en/contact.html`.  
Look at the line `info[at]mp3tag.de for private feedback`.
2. Disallow inline scripts globally and hit refresh. The mail address is gone now (which is correct behaviour).
3. Create local noop-rule for `mp3tag.de` and refresh. The email adress should be visible now, but is still vanished.
4. Refresh the tab with ctrl + F5 and the mail adress will correctly appear without changing any further rules.

### Screenshot after step 3:
![unbenannt](https://user-images.githubusercontent.com/1388389/36356714-eef5cdd6-14f5-11e8-9e45-f070ef054295.PNG)

### Your settings
default settings, advanced user

- OS/version: Win7x32
- Browser/version: Firefox 58.0.2 (32-bit)
- uBlock Origin version: 1.15.6

**Comment by gorhill** [comment]
Duplicate of #3408.

**Comment by stonecrusher** [comment]
Sorry for duplicate, I apparently searched for the wrong keywords like `inline script` or `refresh`
The other issue exclusively uses the term `reload` :-/
